mars/services/task/execution/mars/executor.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         lifecycle_api: LifecycleAPI,
         scheduling_api: SchedulingAPI,
         meta_api: MetaAPI,
+        resource_evaluator,
```

Comment:
Can you add a type for `resource_evaluator`?

Reply:
Yes. Thanks for your reminder.